stock_picking_update_responsible/views/stock_picking_views.xml

```diff
@@ -0,0 +1,15 @@
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form (in Stock Picking Update Resposible)</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='extra']//field[@name='user_id']" position="after">
+                <field name="is_user_id_editable" invisible="1"/>
+            </xpath>
```

Comment:
Could you check if view `readonly` change still works without `invisible` field addition?